src/Resources/config/api_resources/GiftCard.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" ?>
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%setono_sylius_gift_card.model.gift_card.class%" shortName="GiftCard">
```

Comment:
> `shortName="GiftCard"`

Is this the convention for short names? What is it used for? Should it be namespaced?

Reply:
It seems to be the one from Sylius. It is used to display the resource name in API doc
![image](https://user-images.githubusercontent.com/9363039/130732086-295c8a5d-6690-45fe-8076-190fac8c59d6.png)

It should not be namespaced imo. But it seems that by default the name will be `GiftCard` for us. So perhaps not really needed here